src/components/cc-product-card/cc-product-card.js

```diff
@@ -0,0 +1,198 @@
+import { css, html, LitElement } from 'lit';
+import { dispatchCustomEvent } from '../../lib/events.js';
+import { i18n } from '../../lib/i18n.js';
+import '../cc-img/cc-img.js';
+import '../cc-button/cc-button.js';
+import '../cc-badge/cc-badge.js';
+
+/**
+ * @typedef {import('./cc-product-card.types.js').Keyword} Keyword
+ */
+
+/**
+ * A component displaying some information about a product in a card.
+ *
+ * @cssdisplay block
+ *
+ * @event {CustomEvent} cc-product-card:select - Fires the product name when the product is selected.
+ *
+ */
+export class CcProductCard extends LitElement {
+
+  static get properties () {
+    return {
+      description: { type: String },
+      icon: { type: String },
+      link: { type: String },
+      keywords: { type: Array },
+      name: { type: String },
+    };
+  }
+
+  constructor () {
+    super();
+
+    /** @type {string|null} The description of the product */
+    this.description = null;
+
+    /** @type {string|null} The icon of the product */
+    this.icon = null;
+
+    /** @type {string|null} The link where the user will be redirected when the product is selected  */
+    this.link = null;
```

Comment:
nitpick: `link` is more about a visible representation on screen. Here this property is a URL, so maybe we should rename it into `url`.